### PR TITLE
Some clean up in the pubsub registry code.

### PR DESCRIPTION
**Pull request by jessemyers**
There's some nastiness in the registry logic right primarily to handle
legacy and test behaviors. As a first step, remove the registry base
class and clean up the registry interfaces.

### CODE REVIEW

File: microcosm_pubsub/daemon.py [hunk 1]
```diff
@@ -16,7 +16,7 @@ def make_arg_parser(self):
 
     def run_state_machine(self):
         for media_type in self.graph.sqs_message_handler_registry.keys():
-            handler = self.graph.sqs_message_handler_registry[media_type]
+            handler = self.graph.sqs_message_handler_registry.find(media_type)
```
Comment (jessemyers) [inline microcosm_pubsub/daemon.py]: Use `find()` instead of `__getitem__`.

File: microcosm_pubsub/registry.py [hunk 2]
```diff
@@ -16,21 +16,17 @@ class AlreadyRegisteredError(Exception):
...
+    Keeps track of available message schemas.
 
     """
     def __init__(self, graph):
-        """
-        Create registry.
-
-        """
-        self.mappings = dict()
+        self._mappings = dict()
```
Comment (jessemyers) [inline microcosm_pubsub/registry.py]: Ensure that the member variable is internal.

File: microcosm_pubsub/tests/fixtures.py [hunk 3]
```diff
@@ -23,8 +25,16 @@ def name(self):
         return "example"
 
 
+class Foo(object):
```
Comment (jessemyers) [inline microcosm_pubsub/tests/fixtures.py]: Unify fixtures.

**Comment by netama** [comment]
LGTM

**Pull request closed without merging**